.github/workflows/deploy.yml

```
@@ -113,5 +113,5 @@ jobs:
             git pull origin master
 
             docker pull ${{ secrets.DOCKER_USERNAME }}/header-back-img:master
-            docker compose -f docker-compose.yml down -v || true
-            docker compose -f docker-compose.yml up -d --build
\ No newline at end of file
+            docker compose -f docker-compose.yml down || true
+            docker compose -f docker-compose.yml up -d
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)